.github/workflows/ci.yml

```diff
@@ -73,25 +67,24 @@ jobs:
           - ember-lts-3.28
           - ember-lts-4.4
           - ember-lts-4.8
+          - ember-lts-4.12
+          - ember-lts-5.4
           - ember-release
           - ember-beta
           - ember-canary
-          - ember-default-with-jquery
-          - ember-classic
-#          - embroider-safe
-#          - embroider-optimized
+          # - embroider-safe
+          # - embroider-optimized
```

Comment:
IMO does not make sense sync with embroider blueprint and don't support it on CI

Reply:
@villander as this was commented out, I didn't touch. Let me open separate PR and uncomment these scenarios. I'm not sure why they been commented and hat was there error

Reply:
Oh sorry! nevermind

Reply:
@SergeAstapov have you tried enable them to make it compatible with embroider?